compiler/rustc_mir_build/src/builder/scope.rs

```diff
@@ -180,10 +180,10 @@ struct BreakableScope<'tcx> {
 
 #[derive(Debug)]
 struct ConstContinuableScope<'tcx> {
-    /// The if-then scope or arm scope
+    /// The if-then scope or arm scope.
     region_scope: region::Scope,
```

Comment:
is this accurate for us? we just use the `topmost` scope in practice

Reply:
so this is just the innermost scope that the `#[const_continue]` is in I think?

Reply:
This is the scope for the `#[loop_match]` which `#[const_continue]` will jump to.